funnel/assets/cypress/integration/10_addProposal.js

```diff
@@ -56,5 +57,13 @@ describe('Add a new proposal', function () {
     cy.wait('@post-comment');
     cy.get('.comment__body').contains(proposal.proposer_note);
     cy.get('.comment__header').contains(user.username);
+
+    cy.visit('/');
+    cy.logout();
+    cy.wait(1000);
+    cy.login('/' + profile.title, editor.username, editor.password);
+    cy.visit('/updates');
+    cy.wait(1000);
```

Comment:
Instead of this, wait for updates fetch
cy.route('GET', '**/updates').as('fetch-updates');
cy.wait('@fetch-updates');